### ISSUE TITLE
TableReporter gives error if it has 0 inputs.

### ISSUE THREAD

**Issue by chrisdembia** [opensim-org/opensim-core#1770]
The following code (in MATLAB) causes the given error:

```
>> m = Model();
>> r = TableReporter();
>> m.addComponent(r);
>> m.initSystem();
Java exception occurred:
java.io.IOException: Key = labels
	In file
        /Users/chris/repos/opensim/16/opensim-core/OpenSim/Common/AbstractDataTable.h:390
	In function 'setColumnLabels'
	at org.opensim.modeling.opensimSimulationJNI.Model_initSystem(Native Method)
	at org.opensim.modeling.Model.initSystem(Model.java:835) 
```

This is because `AbstractDataTable::setColumnLabels()` does not allow an empty list of column labels. As a consequence, TableReporters with no inputs (no calls to `addToReport()`) generate an exception message. However, I think it's fine for a TableReporter to have no inputs (in this case, nothing would be reported).

@jenhicks reported this bug during the dry run for the TGCS 2017 workshop.

**Comment by tkuchida** [comment]
> I think it's fine for a TableReporter to have no inputs (in this case, nothing would be reported).

Seems like this situation would arise if the user forgot to call `addToReport()`, in which case a warning might be nice. Either solution would be better than a java.io.IOException.

**Comment by tkuchida** [comment]
Fixed by #1771 (merged).
